aten/src/ATen/EmptyTensor.cpp

```diff
@@ -62,6 +62,11 @@ size_t computeStorageNbytes(
     size_t itemsize_bytes,
     size_t storage_offset
   ) {
+  // Similar to a message in as_strided_batching_rule
+  TORCH_CHECK(sizes.size() == strides.size(),
+    "size and stride must have the same length! Got size ", sizes,
+    " and stride ", strides);
```

Comment:
I looked at what happened previously, and I think this is the error message that was raised:
https://github.com/pytorch/pytorch/blob/34bdd46e6e6a15dfa1c765c1fb8d9d7bc02dc102/c10/core/TensorImpl.h#L1442-L1448

Reply:
yep, a similar message is in `new_empty_strided_batching_rule` as well. i'll change it to be consistent